launcher/FileSystem.cpp

```diff
@@ -228,6 +229,9 @@ bool trash(QString path, QString *pathInTrash = nullptr)
 #if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     return false;
 #else
+    // FIXME: Figure out trash in Flatpak. Qt seemlingly doesn't use the Trash portal
```

Comment:
```suggestion
    // FIXME: Figure out trash in Flatpak. Qt seemingly doesn't use the Trash portal
```

:slightly_smiling_face: